hubspot3/tickets.py

```diff
@@ -185,7 +184,7 @@ def _get_recent(
 
                 changes = self._merge_changes_with_history(changes, tickets_history,
                                                            max_time_diff_ms)
-                yield changes
+                yield from changes
 
     def _merge_changes_with_history(self,
```

Comment:
Do we need these two methods now that you're using `yield from`? It's basically a wrapper for the built-in `list`.

Reply:
I was trying to keep the same usability as the other similar methods